dashboard/state_aggregator.py

```diff
@@ -80,7 +82,10 @@ def _convert_filters_type(
         A new list of filters with correct types that match the schema.
     """
     new_filter = []
-    schema = {field.name: field.type for field in fields(schema)}
+    if dataclasses.is_dataclass(schema):
+        schema = {field.name: field.type for field in fields(schema)}
+    else:
+        schema = schema.schema_dict()
```

Comment:
isn't it asdict?

Reply:
no, it's mapping to field -> field type, rather than values.